lib/rdstation.rb

```diff
@@ -2,14 +2,21 @@ module RDStation
   class << self
     attr_accessor :configuration
 
+    HOST = 'https://api.rd.services'.freeze
+
     def configure
       self.configuration ||= Configuration.new
+      self.configuration.base_host = HOST
       yield(configuration)
     end
+
+    def host
```

Comment:
I understood the goal of this method, but I'm not sure if it is ok for this PR.

I noticed the usage of `RDStation.configuration&.method` on every part of this repository, why does only this configuration method have a method here?

Maybe we can keep using the syntax above (via `RDstation.configuration`) and in the future, we can create a better way to access the configuration methods.

What do you think?

Reply:
+1 here
I don't see why we need this. In set the default value in `configure` method, the only way this will break is if the user change that value to nil, correct?
@gil27 WDYT?

Reply:
@angeliski 

It is possible to call a method without calling configure before (at least on specs), since it is possible, we need to check if configuration is nil.

I started doing `RDStation.configuration.base` directly and when I ran the specs, almost every single test broke. So, in order to ensure everything would still work, I've added `host`.


@m3nd3s

The ideia of `host` is to not repeat my self on every single block of code doing this:
```ruby
host = RDStation.configuration&.method || 'https://api.rd.services'
"#{host}/my/url"
```

> Maybe we can keep using the syntax above (via RDstation.configuration) and in the future, we can create a better way to access the configuration methods.

I think by using `host` method will help us in the feature, since we will have this logic in a single place.

Reply:
I got it!

OK, I can reconsider allowing this to pass by the urgency of this feature. We can refactor this in the future.

What do you think @angeliski ? If you would be ok with this I'll resolve this thread.

Reply:
I am thinking 🤔 
Maybe we could define the default `HOST` at the initialize method in the `Configuration` class, and avoid the others null check references?

In this way, everyone will use  `RDStation.configuration.host` and get the updated value (even without call the `configure`)
WDYT?

Reply:
configuration would still be `nil`, It is defined only when we call `configure`.

```ruby
    def configure
      self.configuration ||= configuration.new
      self.configuration.base_host = host
      yield(configuration)
    end
```

We could make configuration always exist, but this would impact other parts where it checks if there is a configuration and since it's not our purpose here, I don't think it would be valid.